### ISSUE TITLE
GNS3 doesn't work with Fedora 31

### ISSUE THREAD

**Issue by medharrak** [GNS3/gns3-server#1688]
Hi , 
I installed GNS3 with Fedora 31 , the issue is that I can't add any node to the created project when I run GNS3 with normal user, but with superuser it works just fine.
=> debug 2   --> shows no errors
{
'Builtin': {
},
'Docker': {
'containers': []
},
'Dynamips': {
'allocate_aux_console_ports': False,
'dynamips_path': '/usr/bin/dynamips',
'ghost_ios_support': True,
'mmap_support': True,
'sparse_memory_support': True
},
'GraphicsView': {
'default_label_color': '#000000',
'default_label_font': 'TypeWriter,10,-1,5,75,0,0,0,0,0',
'draw_link_status_points': True,
'draw_rectangle_selected_item': False,
'grid_size': 75,
'limit_size_node_symbols': True,
'scene_height': 1000,
'scene_width': 2000,
'show_grid': False,
'show_interface_labels': False,
'show_interface_labels_on_new_project': False,
'show_layers': False,
'snap_to_grid': False,
'zoom': None
},
'IOU': {
'iourc_content': '',
'license_check': True
},
'Qemu': {
'enable_kvm': True,
'require_kvm': True
},
'VMware': {
'block_host_traffic': False,
'host_type': 'ws',
'vmnet_end_range': 100,
'vmnet_start_range': 2,
'vmrun_path': ''
},
'VPCS': {
'vpcs_path': ''
},
'VirtualBox': {
'vboxmanage_path': ''
}
}
GET http://127.0.0.1:3080/v2/settings {}
GET http://127.0.0.1:3080/v2/settings {}
GET http://127.0.0.1:3080/v2/settings {}
GET http://127.0.0.1:3080/v2/appliances/templates {}
GET http://127.0.0.1:3080/v2/appliances/templates {}
GET http://127.0.0.1:3080/v2/appliances/templates {}
GET http://127.0.0.1:3080/v2/appliances {}
GET http://127.0.0.1:3080/v2/appliances {}
GET http://127.0.0.1:3080/v2/appliances {}
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
Decoding response from http://admin@127.0.0.1:3080/v2/appliances/templates response 200
Decoding response from http://admin@127.0.0.1:3080/v2/appliances/templates response 200
Decoding response from http://admin@127.0.0.1:3080/v2/appliances/templates response 200
Decoding response from http://admin@127.0.0.1:3080/v2/appliances response 200
Decoding response from http://admin@127.0.0.1:3080/v2/appliances response 200
Decoding response from http://admin@127.0.0.1:3080/v2/appliances response 200
GET http://127.0.0.1:3080/v2/settings {}
GET http://127.0.0.1:3080/v2/settings {}
GET http://127.0.0.1:3080/v2/settings {}
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
GET http://127.0.0.1:3080/v2/settings {}
GET http://127.0.0.1:3080/v2/settings {}
GET http://127.0.0.1:3080/v2/settings {}
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
GET http://127.0.0.1:3080/v2/settings {}
GET http://127.0.0.1:3080/v2/settings {}
GET http://127.0.0.1:3080/v2/settings {}
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
GET http://127.0.0.1:3080/v2/settings {}
GET http://127.0.0.1:3080/v2/settings {}
GET http://127.0.0.1:3080/v2/settings {}
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200
Decoding response from http://admin@127.0.0.1:3080/v2/settings response 200

Any idea please ?

**Comment by joemhiggins** [comment]
I spent 4 hours trying to assist, but I gave up. I was however using a nested Fedora inside the my GNS3 VM on my windows 10 gns3 version 2.10 (which I have never done before). I did successfully load Fedora 31 inside there with whatever version the gns3 documentation says for Fedora - but that is extremely complicated. I only loaded the ethernetswitch in the nested Fedora GNS3 gui and what happens is - exactly what Medharrak says, I give the project a name and when I drag the ethernetswitch into the center of the gui, the ethernetswitch flies back to the all appliances dock. There are more problems I encountered, but my logfiles are 300,000 bytes long, so it is best to say nothing because KVM is so complicated.
